Modules/_testcapimodule.c

```diff
@@ -2385,7 +2385,8 @@ test_null_strings(PyObject *self, PyObject *Py_UNUSED(ignored))
     PyObject *tuple = PyTuple_Pack(2, o1, o2);
     Py_XDECREF(o1);
     Py_XDECREF(o2);
-    return tuple;
+    Py_XDECREF(tuple);
+    Py_RETURN_NONE;
```

Comment:
If `tuple` is NULL, the result should be NULL.

Reply:
I should wait for buildbots to finish to push this change right?

Reply:
No, you do not need to wait for buildbots.